pages/index.html

```diff
@@ -4,9 +4,46 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Wanjun's homepage</title>
+  <link href="styles/style.css" rel=Stylesheet>
 </head>
 <body>
-  
+  <header>
+    <div id="home-header"><h1> WANJUN LAN </h1></div>
+    <div class="image"></div>
+    <nav>
+      <ul class="home-menu">
+        <li><a href="http://127.0.0.1:5502/pages/index.html"><b>Home</b></a></li>
+        <li><a href="http://127.0.0.1:5502/pages/about.html"><b>About</b></a></li>
+        <li><a href="http://127.0.0.1:5502/pages/portfolio.html"><b>Portfolio</b></a></li>
+      </ul>
+    </nav>
+  </header>
+  <div>
+    <div>
+      <div class="index-main-container">
+        <div>
+          <a href="/about.html">
```

Comment:
This link also does not work because it is an absolute path instead of a relative path. Please see the comment above about absolute vs relative.